docker-compose.yml

```diff
@@ -9,22 +19,27 @@ services:
     tty: true
     environment:
       SERVICE_NAME: app
+      DB_CONNECTION: pgsql
+      DB_HOST: postgres
+      DB_PORT: '5432'
+      DB_DATABASE: postgres
+      DB_USERNAME: postgres
+      DB_PASSWORD: postgrespassword
     volumes:
       - ./:/var/www
       - ./services/php/local.ini:/usr/local/etc/php/conf.d/local.ini
     networks:
       - default
     depends_on:
-      mysql:
-        condition: service_healthy
+      - postgres
 
   nginx:
     container_name: nginx
-    image: nginx:alpine
+    image: nginx
     restart: unless-stopped
     tty: true
     ports:
-      - "80:80"
+      - "9001:80"
```

Comment:
any reason why you changed the port number ?

Reply:
For the developer experience of running everything

I thought it would be funny if:
 - App `http://localhost:3000/`
 - Vercel functions `http://localhost:3000/api/hasura/auth`
 - Hasura graphQL `http://localhost:8080/v1/graphql`
 - Laravel `http://localhost/api/v2/users`

Instead
 - Laravel `http://localhost:9000/api/v2/users`

Reply:
![image](https://user-images.githubusercontent.com/193667/147599298-82d4faef-f5f2-42a1-911a-1eaf91ce3dea.png)

Anyway I agree it's a good idea not to use default ports on the system

Reply:
it's common practice to listen to default port 80 for web servers, otherwise you'll need to access website via coordinape.com:9001 . But since we're using docker for local dev only so i guess doesn't matter

Reply:
Yeah absolutely, if this were a production container it'd be a different story